bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -1087,4 +1087,28 @@ public void testSuspendCompaction() throws Exception {
         assertEquals(responseMap7.get("isMajorGcSuspended"), "false");
         assertEquals(responseMap7.get("isMinorGcSuspended"), "false");
     }
+
+    @Test
+    public void testTriggerEntryLocationCompactService() throws Exception {
+        HttpEndpointService triggerEntryLocationCompactService = bkHttpServiceProvider
+                .provideHttpEndpointService(HttpServer.ApiType.TRIGGER_ENTRY_LOCATION_COMPACT);
+
+        //1.  Put, should return OK
+        HttpServiceRequest request1 = new HttpServiceRequest(null, HttpServer.Method.PUT, null);
+        HttpServiceResponse response1 = triggerEntryLocationCompactService.handle(request1);
+        assertEquals(HttpServer.StatusCode.OK.getValue(), response1.getStatusCode());
+        LOG.info("Get response: {}", response1.getBody());
+
+        //2.  GET, should return OK
+        HttpServiceRequest request2 = new HttpServiceRequest(null, HttpServer.Method.GET, null);
```

Comment:
Trigger compaction first and send get request to get the result, and please cover multiple ledger directory case.

Reply:
addressed